src/ripple/rpc/impl/KeypairForSignature.h

```diff
@@ -33,6 +34,7 @@ struct KeyPair
 };
 
 KeyPair keypairForSignature (Json::Value const& params, Json::Value& error);
```

Comment:
It looks like it's _possible_, but overly complicated, and not worth it for such a simple struct. https://stackoverflow.com/questions/26276943/make-custom-type-tie-able-compatible-with-stdtie

However, the more I think about it, the more I think it would be sufficient if generateKeyPair had a descriptive comment in the header and/or implementation: ```
// .first is the public key. .second is the secret key.
```
The first thing someone unfamiliar with the function is going to do is look at the definition anyway, and that comment probably answers the question as quickly as the struct.